src/layouts/core-dev-dot-layout/core-dev-dot-layout.module.css

```diff
@@ -32,6 +32,7 @@
   font-family: var(--font-body) we set on our `body` element needs to be
   re-declared here, so that our "system-ui" font stack cascades as expected. */
 	font-family: var(--font-body);
+	background-color: var(--token-color-surface-primary);
```

Comment:
Adding a background color to the entire layout, lmk if folks disagree here. This way we don't have to go in and set background color on the sidebar and main content areas. But i'm open for that approach also

Reply:
I dig the approach you have here 🙌